### PR TITLE
Create MANUAL_TEST.md

### PR DESCRIPTION
**Pull request by ethansocal**
A task list to remember what to test before submitting the code.

### CODE REVIEW

File: tests/MANUAL_TEST.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Manually Test the Features of WTPython
+
+* Check if it can run with `wtpython example/runs_with_error.py`
+    - Check if pressing `q` and `ctrl + c` exits the app
+    - Check if pressing `left`, `right`, `k`, and `f` change the question
```
Comment (briancohan) [inline tests/MANUAL_TEST.md:5]: should be `l` not `f`

**Review by VagishVela — CHANGES_REQUESTED** [review]
Please update as the keys have changed as of #75

File: tests/MANUAL_TEST.md (hunk 1 above)
Comment (VagishVela) [reply tests/MANUAL_TEST.md:5]: It is now `j` in the main branch.

**Comment by VagishVela** [comment]
I've fixed the issue. Can someone review & merge?

**Review by briancohan — APPROVED** [review]

**Review by VagishVela — APPROVED** [review]
LGTM

**Pull request merged**